modules/ort/src/main/kotlin/workflow/processors/enricher/OrtDownloaderProcessor.kt

```diff
@@ -61,6 +61,13 @@ class OrtDownloaderProcessor : AbstractProcessor() {
             try {
                 LOGGER.debug("Download sources via ORT Downloader for '${name}'")
                 val ortDownloadDirectory = createTempDir("ortDownloaderDirectory").also { it.deleteOnExit() }
+                // In some cases, the protocol of a VcsInfo.url is SSH, which leads to hanging builds,
```

Comment:
I am not sure if this is needed here.

@sschuberth, if  I understood you right, this is exactly what processed already does, right? What about Gitlab and Bitbucket, the same applies for that, or not?

Reply:
It is needed; that's the thing I wanted to discuss with @bs-ondem yesterday.

While the `url` in `vcsProcessed` is normalized from an [scp-like Git URL](https://git-scm.com/docs/git-clone#_git_urls_a_id_urls_a) like [git@github.com:kaelzhang/node-ignore.git](https://github.com/kaelzhang/node-ignore/blob/aa4e6b6bd9a22061364d8816d39ec0c15fc7cd47/package.json#L28) to a standard `ssh`-URL like `ssh://git@github.com/kaelzhang/node-ignore.git`, the normalization deliberately does *not* change the protocol.

It's a common mistake of package maintainers to use a clone URL that requires authorization in meta-data fields that just demand read-only access. Ideally, this should be reported upstream, but this would not fix the meta-data for already released packages. In plain ORT, the correct way to address this issue would be to create a [curation](https://github.com/heremaps/oss-review-toolkit/blob/master/docs/config-file-curations-yml.md) for the package. But for the sake of simplicity, and as it's like said a common mistake, my suggestion was to hard-code a generic "curation" that maps `ssh`- to `https`-URLs here.